src/ol/overlay.js

```diff
@@ -188,6 +194,16 @@ ol.Overlay.prototype.getElement = function() {
 
 
 /**
+ * Get the feature identifier which is set on constructor.
```

Comment:
This is not a *feature* identifier, but an *overlay* identifier. Also, does this need to be `@api`?